setup.py

```diff
@@ -47,7 +47,7 @@
         # grpc.Channel.close() method isn't added until 1.32.0.
         # https://github.com/grpc/grpc/pull/15254
         "grpcio >= 1.32.0, < 2.0dev",
-        "pyarrow >= 1.0.0, < 4.0dev",
+        "pyarrow >= 1.0.0, < 5.0dev",
     ],
     "pandas": ["pandas>=0.23.0", "pyarrow >= 1.0.0, < 4.0dev"],
     "bignumeric_type": ["pyarrow >= 3.0.0, < 4.0dev"],
```

Comment:
This is the explanation we were looking for. :)

This is causing us to get 3 when these extras are requested.

So there _is_ a problem. :)

The snippets hard-code 4 in their requirements.txt. That change was made by renovate-bot  a couple of days ago.

This all seems overly complicated. :(

Reply:
What a bummer. :/

:laughing: